sentry/values.yaml

```diff
@@ -406,6 +406,18 @@ clickhouse:
         replica:
           backup:
             enabled: false
+      users:
+        enabled: false
+        user:
+        # the first user will be used if enabled
+        - name: default
+          config:
+            password: ""
+            networks:
+            - ::/0
```

Comment:
Should we include `0.0.0.0/0` as a default here? The IPv6 rule didn't seem sufficient to allow access when testing locally. (not that users shouldn't put their cluster subnets here anyway, maybe worthy of a comment?)

Reply:
I just copied default values from 
https://github.com/sentry-kubernetes/charts/blob/develop/clickhouse/values.yaml#L324
Also ::/0 usually mean both IPv4 and IPv6 subnetworks
https://clickhouse.tech/docs/en/operations/settings/settings-users/  
```
To open access for user from any network, specify:
<ip>::/0</ip>
```